scss/custom/_modal.scss

```diff
@@ -0,0 +1,73 @@
+//
+// Full Scale Modal
+//
+
+.modal-full {
+  width: 100%;
+  max-width: 100%;
+  height: 100%;
+  padding: 0;
+  margin: 0;
+
+  .modal-content {
+    // Fixed for older phones
+    display: -webkit-flex; /* stylelint-disable-line */
+    display: -moz-box; /* stylelint-disable-line */
+    display: -ms-flexbox; /* stylelint-disable-line */
+    -webkit-flex-direction: column; /* stylelint-disable-line */
+    -moz-flex-direction: column; /* stylelint-disable-line */
+    width: 100%;
+    height: 100%;
+    border: none;
+    border-radius: 0;
+  }
+
+  .modal-header,
+  .modal-footer {
+    // Fixed for older phones
+    display: -webkit-flex; /* stylelint-disable-line */
+    display: -moz-box; /* stylelint-disable-line */
+    display: -ms-flexbox; /* stylelint-disable-line */
+
+    width: 100%;
+    box-shadow: $modal-full-shadow;
+  }
+
+  .modal-body {
+    // Fixed for older phones
+    -webkit-box-flex: 1;   /* stylelint-disable-line */ /* OLD - iOS 6-, Safari 3.1-6 */
+    -webkit-flex: 1;       /* stylelint-disable-line */ /* Safari 6.1+. iOS 7.1+, BB10 */
+    -ms-flex: 1;           /* stylelint-disable-line */ /* IE 10 */
+    flex: 1;               /* NEW, Spec - Firefox, Chrome, Opera */
```

Comment:
We should also be fine with keeping just the flex: 1; since the very old browsers iOS6 is no longer in our supported browsers list